lib/rust/ensogl/component/list-editor/src/lib.rs

```diff
@@ -527,7 +531,7 @@ impl<T: display::Object + CloneRef + Debug> ListEditor<T> {
                 &frp.enable_last_insertion_point,
                 f!([model] (gap, gaps, pos, size, is_dragging, enable_all, enable_last) {
                     let is_close_x = pos.x > -gap && pos.x < size.x + gap;
-                    let is_close_y = pos.y > -gap && pos.y < size.y + gap;
+                    let is_close_y = pos.y > 0.0 && pos.y < size.y;
```

Comment:
why this change? The gap was there to display `+` button when the mouse is cldse to the edge of lit editor. Without this change it will not be visible, right?

Reply:
It is visible, the area is just intentionally made a little smaller. The gap was extending the active bounding box in both X and Y axis. Y axis extension wasn't desirable, as it made the insertions way too easy to do accidentally, especially when attempting to drag the node.

Reply:
Wait, but dragging the node by clicking above / below list editor should be hardly supported / unsupported. I mean, there is just a few pixels for that and assuming someone would be able to find these pixels is bad. I prefer the drag areas to be bigger to easier initiate dragging / insertion of elements. Of course, currently there is problem with node dragging because we don't have the icons on the left, but it is not solvable by making this are smaller IMO.

Reply:
I will adjust that when working on new node design, as we will have slightly more area to work with. For now, it behaves much better that way. It is still not hard to drag the elements, you simply drag the text anywhere. Also, using `gap` to extend drag area vertically was a little surprising to me. I would prefer that to be a separate configuration input.